babel.config.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+module.exports = (api) => ({
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        ...(api.env('test') ? { targets: { node: 'current' } } : {}),
+        corejs: 3.27,
+        useBuiltIns: 'entry',
```

Comment:
Using `useBuiltIns: 'entry'` (and `corejs`) on babel/preset-env would make the InboxSDK include global polyfills. I don't think that would be a good idea for us, and it works against babel/plugin-transform-runtime's corejs option which includes local polyfills.

Reply:
Dropped in [772ce6d](https://github.com/InboxSDK/InboxSDK/pull/836/commits/772ce6d2eb90e8ec47c40d8cd6cd933fe87821f8)

Longer term, I wonder what `transform-runtime` gets us as library (and not a bundle) should MV3 ever become the norm.

https://github.com/babel/babel/issues/10759#issuecomment-628130147 points out some issues around regenerator (which we won't need in Safari 14+ support).

Reply:
babel/plugin-transform-runtime makes it so 1) instead of every JS-produced file including its own definition of helpers like _interopRequireDefault, it's imported from one place, 2) because we use the corejs option with babel/plugin-transform-runtime, it includes local polyfills for some newer JS features. 1 seems generally useful but 2 seems like something we can safely rip out as we have been using the old version of corejs (`"corejs": 2`) and I believe all browsers we target now have natively implemented everything we used to rely on corejs 2 for.

I'd be a fan of this PR cutting things down further by removing the `corejs: 3` option on babel/plugin-transform-runtime and adding `regenerator: false`.

Reply:
[d4a8d8e](https://github.com/InboxSDK/InboxSDK/pull/836/commits/d4a8d8e9e19ce9ef2024e94d7e4e535496de43da) should have `corejs` removed once more.